sdk/src/wallet/account/types/mod.rs

```diff
@@ -203,7 +203,7 @@ pub enum InclusionState {
 /// The output kind enum.
 #[derive(Debug, Clone, Eq, PartialEq, Serialize, Deserialize)]
 pub enum OutputKind {
-    /// Alias output.
+    /// Account output.
     Alias,
```

Comment:
Also update?
And in L221 `"Alias" => Self::Alias,`

Reply:
Good catch

Reply:
Done